app/student.rb

```diff
@@ -2,10 +2,15 @@
 class Student < Person
   def initialize(age, name, parent_permission, classroom)
     super(age, name, parent_permission)
-    @classroom = classroom
+    add_calssrom(classroom)
   end
 
   def play_hooky
     '¯\\(ツ)/¯'
   end
+
+  def add_calssrom(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
As is the convention in ruby, a method that is to make assignment is appended with "=". Kindly implement this.

I also see that you have misspelt classroom in add_calssrom. I recommend that you correct this so that you don't run into unknown errors.